jupyterhub/handlers/pages.py

```diff
@@ -496,7 +496,7 @@ def sort_key(token):
                 continue
             if not token.client_id:
                 # token should have been deleted when client was deleted
-                self.log.warning("Deleting stale oauth token {token}")
+                self.log.warning(f"Deleting stale oauth token {token}")
                 self.db.delete(token)
                 self.db.commit()
```

Comment:
If we are to print a token, lets do it after we are sure we have deleted it.

```suggestion
                self.db.delete(token)
                self.db.commit()
                self.log.warning(f"Deleted stale oauth token {token}")
```

Reply:
The repr of an ORM token object does not include the token value itself, so it's safe to log them. Plus, I think it's usually best to log what you are _about_ to do, because if there's an error you won't see the log message until it succeeds!

Reply:
> The repr of an ORM token object does

Ah okay than this is perfectly fine!